### ISSUE TITLE
what is adam gradient descent

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#246]
Gradient descent is an optimization algorithm that uses the gradient of the objective function to navigate the search space. Gradient descent can be updated to use an **automatically adaptive step size** for each input variable using a decaying average of partial derivatives, called Adam

Adam is a replacement optimization algorithm for stochastic gradient descent for training deep learning models. Adam combines the best properties of the AdaGrad and RMSProp algorithms to provide an optimization algorithm that can handle sparse gradients on noisy problems
<img width="772" alt="Screen Shot 2022-03-12 at 1 33 29 PM" src="https://user-images.githubusercontent.com/5552371/158033944-1c701612-c947-4ad2-8636-93a36a829a6d.png">

**Comment by WatershedXiaolan** [comment]
Adam optimizer involves a combination of two gradient descent methodologies: Momentum: This algorithm is used to accelerate the gradient descent algorithm by taking into consideration the 'exponentially weighted average' of the gradients. Using averages makes the algorithm converge towards the minima in a faster pace.
